validation/src/test/scala/hmda/validation/rules/lar/validity/V250Spec.scala

```diff
@@ -0,0 +1,26 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.{ LoanApplicationRegister, Loan }
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V250Spec extends LarEditCheckSpec with BadValueUtils {
+  property("Income must be numeric and positive") {
```

Comment:
Might be a bit nit-picky (sorry), but these tests are for loan amounts, not incomes.